### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
--- ast.0
+++ ast.1
@@ -1,27 +1,8 @@
 [debug/ast-codegen] # Foo: AST for codegen
 [debug/ast-codegen]   - Module %1 (unused_function.hlt:11:1-31:2)
-[debug/ast-codegen]       | Foo -> declaration::Module %11 <id="Foo">
-[debug/ast-codegen]       | extern_unused1 -> declaration::Function %7 <linkage="public">
-[debug/ast-codegen]       | extern_unused2 -> declaration::Function %8 <linkage="private">
-[debug/ast-codegen]       | private_unused -> declaration::Function %3 <linkage="private">
-[debug/ast-codegen]       | private_used -> declaration::Function %4 <linkage="private">
-[debug/ast-codegen]       | public_unused -> declaration::Function %6 <linkage="public">
-[debug/ast-codegen]       | test_init -> declaration::Function %9 <linkage="init">
-[debug/ast-codegen]       | test_preinit -> declaration::Function %10 <linkage="preinit">
-[debug/ast-codegen]       | x -> declaration::GlobalVariable %5 <linkage="private">
 [debug/ast-codegen]     - ID <name="Foo"> (unused_function.hlt:11:8)
 [debug/ast-codegen]     - statement::Block (unused_function.hlt:11:1-31:2)
-[debug/ast-codegen]     - declaration::Function %3 <linkage="private"> (unused_function.hlt:11:13-14:49)
-[debug/ast-codegen]       - Function <cc="<standard>"> (unused_function.hlt:14:9)
-[debug/ast-codegen]         - ID <name="private_unused"> (unused_function.hlt:14:15)
-[debug/ast-codegen]         - type::Function <flavor="standard"> (unused_function.hlt:14:9) (non-const)
-[debug/ast-codegen]           - type::function::Result (unused_function.hlt:14:10)
-[debug/ast-codegen]             - type::Bool (unused_function.hlt:14:10) (non-const)
-[debug/ast-codegen]         - statement::Block (unused_function.hlt:14:32)
-[debug/ast-codegen]           - statement::Return (unused_function.hlt:14:34)
-[debug/ast-codegen]             - expression::Ctor (unused_function.hlt:14:41) (const)
-[debug/ast-codegen]               - ctor::Bool <value="false"> (unused_function.hlt:14:41)
-[debug/ast-codegen]         - node::None (unused_function.hlt:14:9)
+[debug/ast-codegen]     - node::None %3 (unused_function.hlt:14:9)
 [debug/ast-codegen]     - declaration::Function %4 <linkage="private"> (unused_function.hlt:14:49-17:47)
 [debug/ast-codegen]       - Function <cc="<standard>"> (unused_function.hlt:17:9)
 [debug/ast-codegen]         - ID <name="private_used"> (unused_function.hlt:17:15)
@@ -42,17 +23,7 @@
 [debug/ast-codegen]           - ID <name="private_used"> (unused_function.hlt:14:49-17:47)
 [debug/ast-codegen]         - expression::Ctor (unused_function.hlt:18:12) (const)
 [debug/ast-codegen]           - ctor::Tuple (unused_function.hlt:18:12)
-[debug/ast-codegen]     - declaration::Function %6 <linkage="public"> (unused_function.hlt:21:1)
-[debug/ast-codegen]       - Function <cc="<standard>"> (unused_function.hlt:21:16)
-[debug/ast-codegen]         - ID <name="public_unused"> (unused_function.hlt:21:22)
-[debug/ast-codegen]         - type::Function <flavor="standard"> (unused_function.hlt:21:16) (non-const)
-[debug/ast-codegen]           - type::function::Result (unused_function.hlt:21:17)
-[debug/ast-codegen]             - type::Bool (unused_function.hlt:21:17) (non-const)
-[debug/ast-codegen]         - statement::Block (unused_function.hlt:21:38)
-[debug/ast-codegen]           - statement::Return (unused_function.hlt:21:40)
-[debug/ast-codegen]             - expression::Ctor (unused_function.hlt:21:47) (const)
-[debug/ast-codegen]               - ctor::Bool <value="false"> (unused_function.hlt:21:47)
-[debug/ast-codegen]         - node::None (unused_function.hlt:14:9)
+[debug/ast-codegen]     - node::None %6 (unused_function.hlt:14:9)
 [debug/ast-codegen]     - declaration::Function %7 <linkage="public"> (unused_function.hlt:24:1)
 [debug/ast-codegen]       - Function <cc="extern"> (unused_function.hlt:24:16)
 [debug/ast-codegen]         - ID <name="extern_unused1"> (unused_function.hlt:24:29)
